content/overview/overview/using-cloudgov-paas.md

```diff
@@ -9,26 +9,26 @@ aliases:
+- Beyond the first step above, the rest of the steps are self-service. You can self-manage access for individual team members at either the org or space level.
 
-- The agency would then use cloud.gov to set up a number of online “spaces” required for developing a federal website. The agency team could choose from a list of additional, optional applications that we offer including technical back-end and search capabilities. Once the requested services are available (which happens automatically upon request), the agency can use them from their code.
+- Once your team is comfortable with the CLI and web-based interface for cloud.gov, the team can write scripts to orchestrate more complex processes involving multiple applications and spaces, all built on the Cloud Foundry API.
 
-- After an agency is set up on cloud.gov:
-  - The agency team would set up a Cloud Foundry “org” that correctly references the agency for billing purposes. This would include a quota.
-  - The team would then add “spaces” to represent as many environments as they need (for example: blue-green production, QA and CI, personal development, etc.)
-  - The team would choose from the list of “managed services” we offer and support (for example: memcache, elasticsearch). The team would then provision “service instances” specific to their spaces.
-  - The team would deploy applications and services that they will manage on their own into those same spaces, and bind them to the managed service instances to ensure they can use them.
-  - The team would then bind “routes” to their applications in order to enable traffic from the outside world to reach them.
-  - The team can watch metrics, logs, and events as their service is used. They can redeploy applications to add capabilities or fix problems.
+## Using the cloud.gov PaaS with an 18F team
 
-- Beyond the first step above, the rest of the steps are self-service. The team can self-manage access for individual team members at either the org or space level.
+18F teams that are working with agency partners use cloud.gov to develop and deploy applications. If 18F is delivering a product, the use of cloud.gov is implicit, and we recoup costs from the partner agency. This ensures the least possible friction for our teams. For example, [Every Kid in a Park](https://everykidinapark.gov/) (for the Department of the Interior) and [College Scorecard](https://collegescorecard.ed.gov/) (for the Department of Education) were both developed and launched using cloud.gov.
 
-- Once they’re comfortable with the CLI and web-based UI for cloud.gov, the team can write scripts to orchestrate more complex processes involving multiple applications and spaces, all built on the Cloud Foundry API.
+The cloud.gov team delivered a presentation about [our early experience developing and using cloud.gov](http://www.altoros.com/cflive/gsa-cuts-app-deployment-from-14-months-to-2-3-days-with-cloud-foundry/) at the May 2015 Cloud Foundry conference.
```

Comment:
We also presented at the CF Summit 2016; maybe we should refer to that here as well since it covers subsequent work for compliance?